test/flows/help_if_you_are_arrested_abroad_flow_test.rb

```diff
@@ -0,0 +1,55 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class HelpIfYouAreArrestedAbroadFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup do
+    testing_flow HelpIfYouAreArrestedAbroadFlow
+    @location_slugs = %w[france iran pitcairn-island syria]
+    stub_worldwide_api_has_locations(@location_slugs)
+  end
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: which_country?" do
+    setup { testing_node :which_country? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of answer_three_syria for syria response" do
+        assert_next_node :answer_three_syria, for_response: "syria"
+      end
+
+      should "have a next node of answer_three_british_overseas_territories for british overseas territory response" do
+        assert_next_node :answer_three_british_overseas_territories, for_response: "pitcairn-island"
+      end
+
+      should "have a next node of answer_one_generic for any other response" do
+        assert_next_node :answer_one_generic, for_response: "france"
+      end
+    end
+  end
+
+  context "outcome: answer_one_generic" do
+    setup do
+      testing_node :answer_one_generic
+      add_responses which_country?: "iran"
+    end
+
+    should "render the Iran specific text for Iran" do
+      assert_rendered_outcome text: "Information pack for British nationals imprisoned in Iran"
+    end
+
+    should "render the standard text for other countries" do
+      add_responses which_country?: "france"
+
+      assert_no_match "Information pack for British nationals imprisoned in Iran", @test_flow.outcome_text
+    end
+  end
```

Comment:
There are also missing outcome tests for `transfer_back`. For example, `answer_one_generic` uses a calculator value so we should probably test that if the use picks a country that that doesn't have a transfer back that the content doesn't show.

Also, it's very strange that `iran_downloads` partial doesn't just show the content by default as Iran is not in `COUNTRIES_WITHOUT_TRANFSERS_BACK` so will always show the content.